_encrypted_internal

```diff
@@ -1 +1 @@
+BEGIN KEYBASE SALTPACK ENCRYPTED MESSAGE. keDIDMQWYvVR58B FTfTeDQNHpBnIo4 D6hcv57NMOWhVpM XSAEE8EQg7QFy1B TTWyhglxCbqaNfy 5K3RGswynWUmxUD HOitcxUbaFkjXMs tT3eJ77abceCsv6 JTZQdTfcax9Udu2 JeO82NGFiQY4EM2 VXDl0Y0F7BO7FOd cPtPK6qg6VdNbnP m1V5P541mIbu5bX kbQrxQFboKEG2cg UMCqHISrDXD8KGt j2ZzuwYaREatzfx wQgAI5bUKec1vUA Q54ENFVmH3Mvr28 XEd2I2JwH1QnFjb utpdZ3CsCugWcZM 7SABm2cHfFFgqdT tKEnrDrjKclyBNg VeRUZvpRoswmifk IqXQd98mFQdx2if jPqYHZr3miAi836 ulHZvc4ZXLGnUQo 8xCwfDVoyn1CVM1 0jDcK3ELs2CmLAz JTP4OceqfDtQgpc BphwqJ6AKbvv0o0 vcU8YNqLkLii5zW ngvC6YbLWYgmnSR aduFaJaAxhsOD6X BT6pWc97wguZztC Xm3P95wdiCuiJBd XoLRvTOvDNcbYfi tWPSLWcC4i5nXgh RhFKeVkBZPgxpdl PpakSJAOVp95KoI POMCvEvTPwgUxh1 XmnjJak20lxW5Js nd0jDQX2UJHlyGE BnN8WJdqpMbxxAd vbXx7NhP7fO1dus CKavpSUctEYAQO0 XDsi0FSFyMNYiQ2 Pcz9OhAVfErSXJa TrVFSkPzmjTYZhH KHg024FBnpZUkpk UqkG5QpUketnNXV 7suuA5ArXqTPiQJ bpafkbJhpfYyFWP HyxB5BlqMiWszBa 5f1BgxbqC1VBSOL a1Hs8BEXul7Zmcu 2B378J9CRYly3Jg EpgNY17ZwOQV6ut 99jGLUncQj7H0vi HwKrVlweADOQCjf qixGT1vn69FQK5Y DuJazUorH3J4NuM 4L3GtjHrU7QHk50 4CulbhspzF5QiAy KNuYNlkcwLRsTjO w7x5znXfGephSkm NleLqc4g9zGkTIN BLbDhLt34PobsFb nMp6GRZuRZwKyjC pZGuU8IO28cmJjB hNjQFfmIdmywGyc xqqgpVnrGD0fUoj rmQUUXDz8z6oCVK MprC8iNfjaVqvmm 35oad4MuM9tWumE OXJyw8QsiHnx1Ax RynlbZAsqrS5rru lRDDZ3ieOfX3Hzc cFMrYadMuhX1P53 WP7Hdpe3FJOelJS aeVU6h5v7Nt2hpk b8egPtbin0NGBJN 6RbzE8eaEKMK72k 34vKCnBChR6Z2Zm 6BejHsyPiSh7yao CL7ZY7QXfJ87amZ Vefcw9e8LXGldde aYLtDA3sAivsEyI ttOBEL0AzaHCDE3 Y8F8C4uI7vB0qQW fgknsHiF1SjwVMH PTZJ6Ovbkzrc1Fc 5aMNR3r6Up9nQSB cMOHISpduCyJ4zn VE5neEB6PzxJ5mF LsAZxxV3z9FiWBR e6ZKRP3QVVpYocW WYFG4ItHeY6ykdH tn9IqVStWKfSDhA PIeohVl3YvUbxL6 lMuSa6dwMLF4yy6 N0dzQQQpTgkiSIe 8InyHknOruO5J5K bUwnnt6DZhqOyRT 7vWbWlOw6bYsJhb rHIudspeJ4E9PFw GfNlMZxpuNuDrJF J0yYHWhYds6rhaA ZYhmiYlz1TStSkI ALBncHCMoU1Eh0k jwRAFvhjXxlBSdP xVwyrYwr9ygCTQM uAIHGVTlLLZHVVA LqqgdGKk5ujTELR ew7EDmteVrVVNjY gCCIEJcBpC9Cj15 i9iy6MGHWmJiZev hANV5GDiKgKEJy8 f8cUdjwhDZGdiuA LZiXr8BhB8oiDjW 0cxrhq1WusOn8sD 9pv3ru6nClT207T rvlWa4fxQcyY0qF QR5Wsum0gLA8LV2 mBPhX7ftPR8oL46 ovtb6Qt0m0DkS1C f1Egswf8rrHuYH2 34UlOFfMM3KE4NT MSmel5JaopLTn6W sbPIDawNsoTF9eV DvYwJR3ecRihYo2 itFDgSSoOsDNgeF TXLyWCASsEK5Fd5 6V855C7sj9nUaEx gLwu3H7pUakWHh7 SmF7t4rizi8uTsj 2gBf4rVJA13CRiF dh9TduSmNAJU7HP pkUDMAIHM6BOSoF gFatJwtZCZEowBy lSKvzqyRv. END KEYBASE SALTPACK ENCRYPTED MESSAGE.
```

Comment:
embedded sentry + mixpanel secrets into our existing `_encrypted_internal`. this is nice bc new employees can just decrypt this to have a local `.env` file ready to go, rather than having to copy + paste the sample and ask teammates for values